tests/foreman/ui/test_discoveredhosts.py

```diff
@@ -99,9 +168,16 @@ def test_delete_discovered_host_1(self):
 
         @Assert: Selected host should be removed successfully
 
-        @Status: Manual
-
         """
+        mac = gen_mac(multicast=True, locally=True)
+        hostname = 'mac{0}'.format(mac.replace(':', ""))
+        self._pxe_boot_host(mac)
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.org_name)
+            self.navigator.go_to_discovered_hosts()
+            self.assertIsNotNone(self.discoveredhosts.search(hostname))
```

Comment:
The way delete_entity is re-written will reduce number of lines from tests. But verification/assertion should always be done at test-level. However, I'll make the changes.